examples/opf/experiments/anomaly/temporal/hotgym/description.py

```diff
@@ -317,6 +313,8 @@
         'anomalyParams': {
            'mode': 'likelihood', # pure(=default) / weighted / likelihood
            'slidingWindowSize': 5, # >=0 / None
+           'binaryAnomalyThreshold': None, # None or from (0,1)
+           'computeFn': Anomaly.COMPUTE_RAW,
```

Comment:
I don't think we should add this since it is the default. Feel free to create an example specifically for custom compute functions though.